src/main/kotlin/uk/gov/justice/digital/assessments/controllers/advice/ControllerAdvice.kt

```diff
@@ -155,7 +161,15 @@ class ControllerAdvice {
       "ApiClientForbiddenException for external client ${e.client} method ${e.method} and url ${e.url}: {}",
       e.message
     )
-    return ResponseEntity(ErrorResponse(status = 401, developerMessage = e.message), HttpStatus.FORBIDDEN)
+    return ResponseEntity(
+      ErrorResponse(
```

Comment:
this was incorrectly returning 401 unauthenticated but should be 403 forbidden as the status code